src/control-plane/backend/lambda/api/common/stack-params-valid.ts

```diff
@@ -414,7 +415,7 @@ async function _checkVpcEndpointsForIsolatedSubnets(pipeline: IPipeline, vpcId:
       );
       _validateEndpointsForModules(pipeline, allSubnets, isolatedSubnetsAZ, selectedPrivateSubnet, vpcEndpoints, vpcEndpointSecurityGroupRules);
     } else {
-      const vpcEndpointServices = vpcEndpoints.map(vpce => vpce.ServiceName!);
+      const vpcEndpointServices = vpcEndpoints.map(vpce => vpce.ServiceName!).filter(s => SOLUTION_VPC_ENDPOINTS.includes(s));
```

Comment:
Let's extract below endpoint lists as part of `SOLUTION_VPC_ENDPOINTS`.

[
      'ecr.dkr',
      'ecr.api',
      'ecs',
      'ecs-agent',
      'ecs-telemetry',
    ] for ingestion servers

'emr-serverless', 'glue' for data processing